src/modules/infobox/InfoboxModule.vue

```diff
@@ -123,7 +129,13 @@ export default {
             )
         },
         showContainer() {
-            return this.isList || this.isEdit || this.showElevationProfile || this.isCombo
+            return (
+                this.isList ||
+                this.isEdit ||
+                this.showElevationProfile ||
+                this.isCombo ||
+                this.importOverlay
+            )
         },
```

Comment:
> An improvement here would be to add the check of (this.isList || ...) && this.floatingTooltip it could then be removed in the isList(), isEdit()

I am not sure this will work as isList() and isEdit() used not only in this place

> From my understanding the import overlay should also be implemented as floating tool tip, do you plan do to the floating tool tip in another PR?

we had a discussion with Pascal and he said we should keep it like this for now and then we will see

Reply:
Ok Then it is ok like this for the moment